ToolsQA/Tests/LoginTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text;
+using ToolsQA.Pages;
+using ToolsQA.Selenium_Basics;
+
+namespace ToolsQA.Tests
+{
+    class LoginTest 
+    {
+        private string _username;
```

Comment:
It's better to have credentials in a separate class - you will use them in other test classes as well since logging in is a common case.